### DISCUSSION TITLE
Digital Design and Computer Architecture - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#167] [Announcements]
# Digital Design and Computer Architecture - CS自学指南

CS自学指南

https://csdiy.wiki/%E4%BD%93%E7%B3%BB%E7%BB%93%E6%9E%84/DDCA/

**Comment by Unik-lif** [comment]
paper的阅读量和老师的语速都是比较惊人的，以前计算所请Onur Mutlu来讲存储，其滔滔不绝地讲了连续三个小时，期间滴水未进。
个人感觉如果能够坚持下来不论是知识还是表达能力都会有巨大的提高（逃）

**Comment by qianyouliang** [comment]
支持

**Comment by wah123wah123** [comment]
弱弱问一下，有没有汇编的课程

**Reply by wah123wah123** [reply]
入门的即可

**Reply by Unik-lif** [reply]
我感觉csapp的汇编讲的巨好无比，看的时候把那一章的习题全做完了，不过只有x86，其他的指令集就去看硬软件接口针对不同ISA的书籍吧。此外，也可以看一些逆向的书，会有奇效。

**Reply by dengyunsheng250** [reply]
逆向书籍有哪些推荐？

**Reply by LingkKang** [reply]
@wah123wah123 可以看一下这个课，前半部分是 RISC-V 汇编。

[[完结] 循序渐进，学习开发一个RISC-V上的操作系统 - 汪辰 - 2021春_哔哩哔哩_bilibili](https://www.bilibili.com/video/BV1Q5411w7z5/)

相关资料和 repo 地址都在评论区。

**Comment by RrYusheng** [comment]
弱弱的问一下，听recording需要登陆伯克利账号怎么解决呢

**Comment by songmuhan** [comment]
请问这个lab有测试吗？

**Comment by JayLi1209** [comment]
今天听完了所有25个lecture，前三个和最后一个lecture可以直接跳过。
整体课程非常连贯，顺序安排合理。因为教授讲的特别特别细，在教授讲每个topic前，自己搜一个十几分钟的梗概先了解一下，上课效果会更好。教授讲的很抽象（具体例子比较少），所以听的时候也挺容易走神。值得一提的是，过往的考试习题和答案都在网站上，youtube上也有TA讲的每年期末卷子解析，很良心。

**Comment by SeeingDragon** [comment]
他这Basys 3开发板要165美元啊？好贵啊，还是我找错了

**Reply by Unik-lif** [reply]
要这个价钱的，不过国内用.edu邮箱可以申请学生discount，可以降价到750块钱附近

**Reply by fetch150zy** [reply]
默默问一下这个学生优惠750是在哪买的，淘宝上这个迪芝伦优惠后还要1200，麻了

**Comment by ccvcpy** [comment]
官方网站购买baysus板子没有国内大陆的地区，没有板子就做不成lab。
是否有人知道国内购买Xilinx Artix-7 FPGA (XC7A35T-1CPG236C)的渠道？

**Reply by ghost** [reply]
淘宝有个叫迪芝伦的店铺，不知道是不是官方

**Comment by tuyula** [comment]
这门课有人学么, 加我微信tuyu550574002
